terraform-modules/teleport-aws-discovery/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+    tls = {
+      source  = "hashicorp/tls"
+      version = "~> 4.0"
+    }
+    teleport = {
+      source  = "terraform.releases.teleport.dev/gravitational/teleport"
+      version = "19.0.0-dev"
+    }
```

Comment:
this should be 18.5 or whatever instead, yeah?

Reply:
when I backport to branch/v18 I'll set it to `>= 18.x.y` for whatever release contains the provider updates we backported recently

Reply:
updated to >= 18.5.1 since we released integration tf support in that version